toolbox/utils.py

```diff
@@ -96,6 +97,27 @@ def get_member_color(member: hikari.Member) -> hikari.Color:
     return hikari.Color(0)
 
 
+def get_user_possessive(user: t.Union[hikari.User, hikari.Member]) -> str:
```

Comment:
```suggestion
def get_possessive(user: t.Union[hikari.User, hikari.Member]) -> str:
```
Seems fairly redundant for it to say `user` when the only argument is a user?

Reply:
I like how explicit the type is, even though it's more verbose then it has to be

Reply:
Nvm that's not what you meant lol. +1 for this

Reply:
This is consistent with `get_member_color`.

Reply:
Fair. Let's leave it how it is then.

Reply:
> This is consistent with `get_member_color`.

The reason it has `member` there is it is ambigious, since roles can have colors too, among other things. But I'm pretty sure the only thing that could have a posessive is a user.

Reply:
There is potentially a case to be made for updating both, as the member in get member colour could be seen as redundant.

Reply:
> > This is consistent with `get_member_color`.
> 
> The reason it has `member` there is it is ambigious, since roles can have colors too, among other things. But I'm pretty sure the only thing that could have a posessive is a user.

Ah, that does make sense actually.

Reply:
I believe it should stay consistent for now. If we want to update both that can be a different PR.

Reply:
After Hyper's clarification I'm actually on the side of making the change now personally. If Hyper still is too I'll update it.